src/Impostor.Api/Events/Game/Player/IPlayerVotedEvent.cs

```diff
@@ -0,0 +1,12 @@
+using Impostor.Api.Net.Inner.Objects;
+
+namespace Impostor.Api.Events.Player
+{
+    public interface IPlayerVotedEvent : IPlayerEvent
+    {
+        /// <summary>
+        ///     Get the player he voted for.
+        /// </summary>
+        IInnerPlayerControl? VotedFor { get; }
```

Comment:
Add VoteType enum and property: Vote, Skip, or None

If VoteType is skip or none, VotedFor is null.

_(alternative enum name is good as well)_